3/2-Datamodel.md

```diff
@@ -0,0 +1,144 @@
+
+## Section 1: Datamodel
+<a name="top"/>
+
+Central to the OpenCannabis project's goals is a *unified datamodel* for use across functional boundaries. When
+cultivators talk about *flowers* with labs, and labs talk about *test results* with retailers, and so on, they are all
+talking about the same thing - but in today's legal cannabis industry, they aren't speaking the same language.
```

Comment:
the _Datamodel_, in _Section 1_, is the core of the spec - it's a model with which cannabis information may be expressed and understood for the purposes of storage, transmission, and computation.